ushahidi/platform-client
# Adds warnings in  case of identical options in checkbox

This pull request makes the following changes:
- Disabled the `Add & Close` if duplicate options are present in the survey. 
- While creating a post for the survey where duplicate options are present
         - If user is admin: warning is shown.
         - Else: only unique options are made visible.

Testing checklist:
- Create a new survey by settings->surveys->create new survey.
- Add a new field "Checkbox".
- [x] While adding two similar field options the add and close button should be disabled.

- Create a new post in a survey which has two similar options.
- [x] If you are logged in as admin you would see a warning.
- [x] If you aren't logged in as admin you would see checkboxes and duplicates won't be present
- [x] No error in console about ng-repeat


Some Screenshots:

While creating survey:
![image](https://user-images.githubusercontent.com/70752431/95746896-2720b180-0cb5-11eb-8268-d2c98eb4b712.png)

While creating post in a survey with two similar options:
![image](https://user-images.githubusercontent.com/70752431/95746956-3d2e7200-0cb5-11eb-8f90-1c1b958bc4d4.png)

After logging out:
![image](https://user-images.githubusercontent.com/70752431/95746979-43bce980-0cb5-11eb-8a3f-e838be76fda5.png)

Fixes ushahidi/platform#3994 .

Ping @ushahidi/platform
